Gemfile

```diff
@@ -48,6 +48,8 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rspec-rails', '~> 3.6'
+  gem "factory_girl_rails"
```

Comment:
It is good practice to add the version of the gem, so you make sure that other people in the future use the same version and don´t break things